subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/page_object/KeyboardElement.kt

```diff
@@ -9,7 +9,7 @@ import androidx.test.platform.app.InstrumentationRegistry
 import org.hamcrest.MatcherAssert.assertThat
 import org.hamcrest.Matchers.`is`
 
-class KeyboardElement : PageObject() {
+class KeyboardElement {
```

Comment:
LSP violation.
It has never conformed `PageObject` contract. It was impossible to create a child element.
Thus, see no reasons to pretend to be `PageObject`.